earthfile2llb/interpreter.go

```diff
@@ -1127,6 +1128,18 @@ func (i *Interpreter) handleSaveImage(ctx context.Context, cmd spec.Command) err
 		fmt.Fprintf(os.Stderr, "Deprecation: using SAVE IMAGE with no arguments is no longer necessary and can be safely removed\n")
 		return nil
 	}
+
+	labels := make(map[string]string)
+	labels["dev.earthly.version"] = version.Version
+	labels["dev.earthly.git-sha"] = version.GitSha
+	labels["dev.earthly.built-by"] = version.BuiltBy
+	labels["dev.earthly.default-installation-name"] = version.DefaultInstallationName
+	labels["dev.earthly.default-buildkitd-image"] = version.DefaultBuildkitdImage
+	err = i.converter.Label(ctx, labels)
+	if err != nil {
+		return i.wrapError(err, cmd.SourceLocation, "save image (label)")
```

Comment:
Our errors tend to start with a `failed to ...` message.

maybe something like this instead:
```suggestion
		return i.wrapError(err, cmd.SourceLocation, "failed to create dev.earthly labels during save image")
```